src/components/contactForm/ContactForm.svelte

```diff
@@ -69,4 +163,17 @@
 		color: black;
 		cursor: not-allowed;
 	}
+
+	.error {
+		display: block;
+		background-color: crimson;
```

Comment:
This has a bit of a low contrast ratio.  https://contrast-ratio.com/#black-on-crimson
Using a light background and dark text colour shows up better https://contrast-ratio.com/#%236b0000-on-%23ffefef